src/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+
+add_executable(app main.cpp
```

Comment:
this looks like an extra newline :wink:

Reply:
See f24ecb6af7205aadb1f19fd1e34be27d9e50fc4a

Reply:
:+1: